### PR TITLE
JENA-1901 Provide default implementations for some of SecurityEvaluator's methods

### PR DESCRIPTION
**Pull request by strangepleasures**

### CODE REVIEW

File: jena-permissions/src/test/java/org/apache/jena/permissions/SecurityEvaluatorTest.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+
+    @Test
+    public void testAnyLogicForTripleOperations() {
+        assertFalse(evaluator.evaluateAny(PRINCIPAL, ALL_ACTIONS, GRAPH, TRIPLE));
+        assertTrue(evaluator.evaluateAny(PRINCIPAL, ALLOWED_ACTIONS, GRAPH, TRIPLE));
+        assertFalse(evaluator.evaluateAny(PRINCIPAL, DISALLOWED_ACTIONS, GRAPH, TRIPLE));
+        assertTrue(evaluator.evaluateAny(PRINCIPAL, EnumSet.allOf(Action.class), GRAPH, TRIPLE));
+    }
+}
```
Comment (afs) [inline jena-permissions/src/test/java/org/apache/jena/permissions/SecurityEvaluatorTest.java:94]: Trivial: missing final newline.

File: jena-permissions/src/main/java/org/apache/jena/permissions/SecurityEvaluator.java [hunk 2]
```diff
@@ -213,7 +210,7 @@
 	 * @throws AuthenticationRequiredException
 	 *             if user is not authenticated and is required to be.
 	 */
-	public boolean evaluate(Object principal, Action action, Node graphIRI)
+	boolean evaluate(Object principal, Action action, Node graphIRI)
```
Comment (afs) [inline jena-permissions/src/main/java/org/apache/jena/permissions/SecurityEvaluator.java:213]: Minor: "public" is used elsewhere on interface methods.

File: jena-permissions/src/test/java/org/apache/jena/permissions/SecurityEvaluatorTest.java (hunk 1 above)
Comment (strangepleasures) [reply jena-permissions/src/test/java/org/apache/jena/permissions/SecurityEvaluatorTest.java]: fixed

File: jena-permissions/src/main/java/org/apache/jena/permissions/SecurityEvaluator.java (hunk 2 above)
Comment (strangepleasures) [reply jena-permissions/src/main/java/org/apache/jena/permissions/SecurityEvaluator.java]: fixed

**Pull request merged**